lib/authorizer/digest.js

```diff
@@ -140,7 +140,8 @@ _extractField = function (string, regexp) {
  */
 function _getDigestAuthHeader (headers) {
     return headers.find(function (property) {
-        return (property.key.toLowerCase() === WWW_AUTHENTICATE) && (_.startsWith(property.value, DIGEST_PREFIX));
+        return (property.key.toLowerCase() === WWW_AUTHENTICATE) &&
+        (_.startsWith(property.value.toLowerCase(), DIGEST_PREFIX.toLowerCase()));
```

Comment:
@coditva  We are using `DIGEST_PREFIX` to generate the `Authorization` header. Changing its value would result in backward incompatibility. 
For instance, if users are expecting `Digest` in their server, but we start sending `digest`, it will break for them.  Even though servers ideally should accept both variants, we cannot guarantee this.